content/cloud-computing/concepts/types-of-cloud/types-of-cloud.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: 'Types of Cloud'
+Description: 'There are three types of cloud computing implementations.'
```

Comment:
```suggestion
Description: 'The types of cloud computing implementations include public, private, hybrid, and multicloud.'
```

What do we think about this possible `Description`? It is brief, and includes many of the major words/phrased used in the content.